manifests/init.pp

```diff
@@ -183,112 +178,100 @@
+  Optional[Variant[Numeric, String]] $package_apt_pin              = undef,
+  String $package_ensure                                           = $rabbitmq::params::package_ensure,
+  String $package_gpg_key                                          = $rabbitmq::params::package_gpg_key,
+  String $package_name                                             = $rabbitmq::params::package_name,
+  Optional[String] $package_source                                 = undef,
+  Optional[String] $package_provider                               = undef,
+  Boolean $repos_ensure                                            = $rabbitmq::params::repos_ensure,
+  Boolean $manage_python                                           = $rabbitmq::params::manage_python,
+  String $rabbitmq_user                                            = $rabbitmq::params::rabbitmq_user,
+  String $rabbitmq_group                                           = $rabbitmq::params::rabbitmq_group,
+  Stdlib::Absolutepath $rabbitmq_home                              = $rabbitmq::params::rabbitmq_home,
+  Integer $port                                                    = $rabbitmq::params::port,
+  Boolean $tcp_keepalive                                           = $rabbitmq::params::tcp_keepalive,
+  Integer $tcp_backlog                                             = $rabbitmq::params::tcp_backlog,
+  Optional[Integer] $tcp_sndbuf                                    = undef,
+  Optional[Integer] $tcp_recbuf                                    = undef,
+  Optional[Integer] $heartbeat                                     = undef,
+  Enum['running', 'stopped'] $service_ensure                       = $rabbitmq::params::service_ensure,
+  Boolean $service_manage                                          = $rabbitmq::params::service_manage,
+  String $service_name                                             = $rabbitmq::params::service_name,
+  Boolean $ssl                                                     = $rabbitmq::params::ssl,
+  Boolean $ssl_only                                                = $rabbitmq::params::ssl_only,
+  Optional[Stdlib::Absolutepath] $ssl_cacert                       = undef,
+  Optional[Stdlib::Absolutepath] $ssl_cert                         = undef,
+  Optional[Stdlib::Absolutepath] $ssl_key                          = undef,
+  Optional[Integer] $ssl_depth                                     = undef,
+  Optional[String] $ssl_cert_password                              = undef,
+  Integer[1, 65535] $ssl_port                                      = $rabbitmq::params::ssl_port,
+  Optional[String] $ssl_interface                                  = undef,
+  Integer[1, 65535] $ssl_management_port                           = $rabbitmq::params::ssl_management_port,
+  Integer[1, 65535] $ssl_stomp_port                                = $rabbitmq::params::ssl_stomp_port,
+  Enum['verify_none','verify_peer']$ssl_verify                     = $rabbitmq::params::ssl_verify,
```

Comment:
I guess it's not a syntax error (tests pass) and just a nitpick, but there's a 'space' missing between the type and `$ssl_verify`.

Reply:
I'm going to approve anyway and leave it to you.

Reply:
fixed